pkg/catalogv2/content/translate.go

```diff
@@ -6,6 +6,19 @@ import (
 	"helm.sh/helm/v3/pkg/repo"
 )
 
+// TranslateURLs - modifies the URLs of the chart versions
```

Comment:
It is unclear how it achieves the required functionality. How and what does it change ? the URL ?